packages/vscode-mdx/.github/README.md

```diff
@@ -0,0 +1,15 @@
+# VS Code Extension
+
+## Contributing
+
+- Clone the repository
+- Run `npm install`
+- Run `npm start`
+- Run `cd packages/vscode-mdx`
+- Run `npm install`
+- Open the VS Code Run and Debug pane and click Run Extension up at the top
+  - Edit MDX files in the `demo` directory that opens
+  - Place and hit breakpoints at desired code sites - the debugger will break
+    (This extends to files in `language-server` as well.)
+- Use the Output channel named Markdown Language Server in the main VS Code
+  window (not the Extension Host window) to monitor LSP communication
```

Comment:
The contributing guidelines are inferred from the [`mdx-js/.github`](https://github.com/mdx-js/.github) repo. This links to https://mdxjs.com/community/contribute/, which contains instructions specifically for the [`mdx-js/mdx`](https://github.com/mdx-js/mdx) repo. I agree this is not ideal.

I suppose there are two things to do here:

1. Have the organization contributing guidelines point to something more generic (we have more repositories). @mdx-js/members thoughts?
2. Write specific contributing guidelines for this project. This project has some specific instructions that don’t apply to other projects, like how to debug a VSCode extension, or how to run the Monaco editor demo.

---

This document is not related to the changes in this PR though. I also don’t think we’ll end up with a file named `.github/README.md`. Could you remove it?

Reply:
I think #2 is better but that's from my perspective as a drive by contributor so it might not be the best choice for the org. My angle here is that when a user of and of the packages has an issue and wants to contribute, familiarizing with the structure of the whole org and stuff is friction. Of course that may be my laziness speaking but most of the time when I contribute to tools I use it is to fix a pain point of my and only if I like the process and feedback think about the org and whether I want to be involved more holistically. Again not sure if this is wise to encourage this but that's where I'm at. 

BTW I created the `.github/README.md` over the normal README because that is the extension marketplace README right? I didn't think build instructions were a fit for that. The dot github readme (sorry I'm on my phone) is a feature of Github which allows replacing what readme renders for the directory which seemed appropriate. Do you want me to move the contribution instructions to the real readme or yeet them altogether?

Reply:
Options 1 and 2 don’t exclude each other. I think we should to both.

`README.md` will indeed be rendered in the extension marketplace, so it’s good to keep this somewhere else. Typically something like this exists in a file named `CONTRIBUTING.md`.

Reply:
Ah okay, I see what you mean. Let me just remove all of the README changes I made, you have a better handle on how to document the contributing guidelines.